api/release_notes/2019-release-notes.md

```diff
@@ -4,6 +4,24 @@ title: 2019 Release Notes
 
 # 2019 Reactor Release Notes
 
+## 2019-09-04
+
+### 1) Technical Accounts can use Product Profiles
+
+When we first released the Launch API, Adobe I/O integrations were not fully integrated into Adobe's Admin Console where regular user accounts are controlled.  As a result, these integrations had a different set of available permissions.
+
+Adobe I/O integrations can now use the same permission groups (called Product Profiles) - created and managed in Adobe's [Admin Console](https://adminconsole.adobe.com/) - that regular users use.
+
+When you login to the Adobe I/O console to create a new integration (or update an existing integration), instead of the older role selector, you'll now see a list of product profiles from the Admin Console.  You'll need to select the product profile you want the integration to belong to.
+
```

Comment:
New integrations will use product profiles.

Legacy integrations will continue to use role to determine authorization (so we don't break existing integrations).
Role will not be visible.
In January legacy integrations will stop using role and will start using any product profiles that are assigned. The problem is this is not verifiable before that time.

It is recommended to replace all existing integrations with new integrations before January to prevent broken integrations at that time.

Reply:
Added this in a less detailed form.